Release non-prerelease version

Hi, thanks for this package - does exactly what I needed, and meant I didn't need to write the same thing!

I just have one request - please could you do a 1.0.0 release (i.e. not alpha)? Currently, if you try and reference this package from another (non pre-release) library you'll get warnings when packing NuGet packages: 

```
C:\Program Files\dotnet\sdk\2.1.302\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5104: A stable release of a package should not have a prerelease dependency. Either modify the version spec of dependency "Kralizek.Extensions.Configuration.AWSSecretsManager [1.0.0-alpha-2, )" or update the version field in the nuspec.
```

Thanks! 🙂

--- a/src/Kralizek.Extensions.Configuration.AWSSecretsManager/Kralizek.Extensions.Configuration.AWSSecretsManager.csproj
+++ b/src/Kralizek.Extensions.Configuration.AWSSecretsManager/Kralizek.Extensions.Configuration.AWSSecretsManager.csproj
@@ -6,7 +6,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>Kralizek.Extensions.Configuration.AWSSecretsManager</PackageId>
     <Description>AWS Secrets Manager configuration provider implementation for Microsoft.Extensions.Configuration.</Description>
-    <PackageVersion>1.0.0-alpha-2</PackageVersion>
+    <PackageVersion>1.0.0</PackageVersion>
     <PackageLicenseUrl>https://raw.githubusercontent.com/Kralizek/AWSSecretsManagerConfigurationExtensions/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Kralizek/AWSSecretsManagerConfigurationExtensions</PackageProjectUrl>
     <PackageTags>dotnet-standard;aws;aws-secrets-manager;</PackageTags>

